shaper-core/src/main/kotlin/dev/icerock/tools/shaper/core/YamlConfigReader.kt

```diff
@@ -63,13 +63,14 @@ object YamlConfigReader {
         val filesList = files as? List<Map<String, Any>>
         val filesDirectory = files as? String
         return filesList?.map { fileMap ->
-            val templateFile = File(directory, fileMap["contentTemplateName"] as String)
+            PathLocator.set(fileMap["contentTemplateName"] as String, directory.absolutePath)
             TemplateConfig.FileConfig(
                 pathTemplate = fileMap["pathTemplate"] as String,
-                contentTemplateName = templateFile.path,
+                contentTemplateName = fileMap["contentTemplateName"] as String,
```

Comment:
save this path to variable, not duplicate getByKey access please

Reply:
Done